CHANGELOG.md

```diff
@@ -16,7 +16,7 @@
 ### Backward-incompatible changes [experimental]
 
 ### Performance enhancements
-* `skbio.tree.nj` wall-clock time was decreased about 66% by vectorizing `skbio.tree._nj._compute_q`. ([#1512](https://github.com/biocore/scikit-bio/pull/1512))
+* `skbio.tree.nj` wall-clock time was decreased about 99% by vectorizing `skbio.tree._nj._compute_q` and avoiding unnecessary recalculations. ([#1512](https://github.com/biocore/scikit-bio/pull/1512), [#1513](https://github.com/biocore/scikit-bio/pull/1513))
```

Comment:
Yes. For a 500x500 matrix, while the switch to `.data` produced a measurable speedup in `nj(dm)` (taking runtimes reported by `%timeit -n 10 tree = nj(dm)` from 1.39 seconds to 949 milliseconds), [I presume that] the majority of the speedup was realized by vectorization (taking the post-#1512 runtime of 115 seconds down to 1.39 seconds).

This said, returns from these changes are non-linear. I have some notes from a now-unknown midpoint of tinkering which suggest returns _increase_ with increasing size of the distance matrix: a 100x100 matrix saw a 77% speedup while a 500x500 matrix saw a 98% speedup.

Unfortunately, for my own data's complete 2831x2831 distance matrix, the current-best runtime (from `%timeit -n 1 tree = nj(dm)`) is still 6m 56s.

I'm happy with the improvements so far for my own wet-bench purposes, so I'm open to any and all suggestions as to how to word the statement in `CHANGELOG.md`; to make it easy, we could just drop the number and say "...was decreased. ([refs to PRs])".